README.md

```diff
@@ -1,14 +1,20 @@
+# Microsoft SARIF Website
 
-# Contributing
+The [Microsoft SARIF website](https://sarifweb.azurewebsites.net) provides resources for the Static Analysis Results Interchange Format
+(SARIF), an open standard log file specification for storing data produced by static analysis tools.
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+## Find Links to Resources
+- SARIF's OASIS open standard
+  - Specification
+  - Tutorials
+  - GitHub project
+- Tools and libraries for developers
+  - Microsoft's open source SARIF SDK for .NET applications
+  - NuGet package for third-party Visual Studio extension interop
+  - Python module
```

Comment:
>Python module [](http://example.com/codeflow?start=4&length=13)

should we also add sarif-js-sdk?

Reply:
Sure but I don't see that linked on the site. Where does it live?

Reply:
NM found it. I'll add this to the site